apps/src/sites/code.org/pages/public/teacher-dashboard/index.js

```diff
@@ -259,6 +259,10 @@ function main() {
       }
     );
 
+    $scope.teacher_managed_section = function (section) {
+      return ['email', 'word', 'picture'].indexOf(section.login_type) > -1;
```

Comment:
that's exactly right, this gets polyfilled by babel. ES6 is safe to use anywhere in apps/ and nowhere outside apps.